src/scripts/charmc

```diff
@@ -389,7 +389,7 @@ filterLegacyLB()
 printVersion()
 {
 	findCharmBin $0
-	version=`cat $CHARMBIN/../include/VERSION`
+	version=`cat $CHARMBIN/../include/charm-version.h`
```

Comment:
This prints out the entire define line.

Reply:
Fixed, thanks. With the most recent commit in this PR, this prints the API version, not the "git" version (as previously). f you want, we can change that.

Reply:
That sounds fine to me, so long as the preamble when running a program still prints out the "git" version.